sdk/clientcore/core/src/main/java/io/clientcore/core/instrumentation/LibraryInstrumentationOptions.java

```diff
@@ -26,7 +26,7 @@
 public final class LibraryInstrumentationOptions {
     private final String libraryName;
     private String libraryVersion;
-    private String schemaUri;
+    private String schemaUrl;
```

Comment:
Schema URL is formally defined in OTel - https://opentelemetry.io/docs/specs/otel/schemas/#schema-url, I'd prefer to keep it here too